modules/nw-rfhe-introduction.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+:_content-type: CONCEPT
+[id="nw-rfhe-introduction_{context}"]
+= How Redfish hardware events work
+The Baremetal Hardware Event Proxy enables applications running on baremetal clusters to respond quickly to Redfish hardware changes and failures such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure. These hardware events are delivered over a reliable low-latency transport channel based on Advanced Message Queuing Protocol (AMQP). The latency of the messaging service is in the 10~20 millisecond range.
+
+The hardware event proxy provides a publish-subscribe service for the hardware events, where multiple applications can use REST APIs to subscribe and consume the events.
+The hardware event proxy supports hardware compliant with Redfish OpenAPI v1.8 or higher.
+
+[id="rfhe-elements_{context}"]
+== Redfish hardware events data flow
+The following figure illustrates an example Redfish hardware events data flow. vDU is used here as an example of an application interacting with Redfish events:
+
+.Redfish hardware events data flow
+image::211_OpenShift_Redfish_dataflow_0222.png[Redfish hardware events data flow]
+
+=== Operator-managed pod
+The Operator makes use of custom resources to manage the pod containing the hardware event proxy and its components using the HardwareEvent CR.
+
+=== Hardware event proxy
+A container created to handle Redfish events. At startup, the hardware event proxy queries the Redfish API and downloads all the Message Registries, including custom registries. The hardware event proxy then begins to receive subscribed events from the Redfish hardware.
+
+The proxy enables applications running on bare-metal clusters to respond quickly to Redfish hardware changes and failures such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure, reported using the `HardwareEvent` CR. 
```

Comment:
I don't think this paragraph needed here, and the last half sentence is not accurate.

Reply:
Done.